api/api-doc/storage_service.json

```diff
@@ -2864,6 +2864,14 @@
                      "allowMultiple":false,
                      "type":"string",
                      "paramType":"query"
+                  },
+                  {
+                     "name":"await_completion",
+                     "description":"Set true to wait for the repair to complete. Set false to skip waiting for the repair to complete. When the option is not provided, it is default to false.",
```

Comment:
s/it is default to false/it defaults to false/

Reply:
yes. updated